components/shared/Testimonial.tsx

```diff
@@ -1,11 +1,12 @@
 import { FC } from "react";
 
 import { createElementSmartLink, createItemSmartLink } from "../../lib/utils/smartLinkUtils";
-import { contentTypes,Testimonial } from "../../models";
+import { contentTypes } from "../../models";
+import { Fact } from '../../models/content-types/fact';
 import { PersonHorizontal } from "./PersonHorizontal";
 
 type Props = Readonly<{
-  item: Testimonial;
+  item: Fact;
 }>;
 
 export const TestimonialComponent: FC<Props> = props => {
```

Comment:
This component doesn't seem to be used. Do we still need it here?

Reply:
Deleting